.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,30 @@
+name: publish gh-pages docs
+on:
+  push:
+    branches:
+      - master
+jobs:
+
+  publish:
+    name: publish docs
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
+    steps:
+    - name: checkout
+      uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: install and build
+      run: |
+        python -m pip install --upgrade pip setuptools
+        pip install .r requirements.txt
+        mkdocs build
+    - name: Deploy
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        publish_dir: ./build-docs
```

Comment:
mokdocs outputs to `./dist` atm not `./build-docs` so this (or the mkdocs.yml) will need to be changed